cmp/grid/GridModel.js

```diff
@@ -789,7 +789,7 @@ export class GridModel {
     //-----------------------
     // Implementation
     //-----------------------
-    async internalAutosizeAsync(colIds, {showMask}) {
+    async internalAutosizeAsync(colIds, {showMask, fillMode}) {
         await start(async () => {
             if (showMask) {
```

Comment:
I think at this point we probably want to move the entire body of this method (with perhaps the exception of the masking) in to an async method on the service.   Easier to have the "brains" of the operation in one place -- e.g. for testing, or swapping in alternative approaches -- and I don't think given the name of the service, it is that terrible for it to actually operate on the the gridModel and the agAPI